prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR_to_HL7/elims_47_04608717_11184_mega_specimen.hl7

```diff
@@ -0,0 +1,20 @@
+MSH|^~\&#|STARLIMS.CDC.Stag^2.16.840.1.114222.4.3.3.2.1.2^ISO|CDC Atlanta^11D0668319^CLIA|CDPH CA CALREDIE^2.16.840.1.114222.4.3.3.10.1.1^ISO|CDPH_CID^2.16.840.1.114222.4.1.2.14104^ISO|20230802180802-0400||ORU^R01^ORU_R01|3015894743_04608717_11184|T|2.5.1|||NE|NE|USA||||PHLabReport-NoAck^PHIN^2.16.840.1.113883.9.11^ISO
+SFT|CDC^^^^^CDC&2.16.840.1.114222.4&ISO^XX^^^CDC CLIA|ELIMS V11|STARLIMS|Binary ID unknown
+PID|1||PID03707032^^^STARLIMS.CDC.Stag&2.16.840.1.114222.4.3.3.2.1.2&ISO^PI~PID123^^^SPHL-000048&2.16.840.1.114222.4.1.10765&ISO^PI||Mega^HL7^MI^^^^L||0000|F|||^^^^^USA^H
```

Comment:
PID-7 here is `0000` and is not preserved in the output, seems like an invalid value we're getting, but just to check, has that been either discussed with ELIMS or added to the spreadsheet?

Reply:
I'll double check but I believe I noted that down (and I think was an issue in an earlier message)

Reply:
Ah yea that's right, in message 40 it was this same issue. Not sure if we ever discussed that with ELIMS or not, but it's preserved when the value is valid like it is in message 52, and I think we assumed we don't need to preserve invalid/meaningless inputs there, which still seems reasonable.